build/NuSpecs/ProjectReunion-Nuget-Native.props

```diff
@@ -0,0 +1,4 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)\Microsoft.ApplicationModel.Resources.props"/>
```

Comment:
><Import Project="$(MSBuildThisFileDirectory)\Microsoft.ApplicationModel.Resources.props"/> [](start = 2, length = 90)

This import is needed even in the non-native case.

Reply:
the native one is not touched

Reply:
not for FWP

Reply:
if I now install the Foundation nuget package in a C# app, the MRTCore props file won't be imported

Reply:
do you have any specific property that needs set?

Reply:
turns out that there is nothing in the MRTCore props file that's needed for C# currently. I didn't realize. ok, it's fine then